README.md

```diff
@@ -0,0 +1,216 @@
+# ParchMint
```

Comment:
This was just imported from the `master` branch but some additional instructions were added after Line 165 as standard text to add to all repos